packages/docs-app/src/examples/icons-examples/iconExample.tsx

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { BaseExample } from "@blueprintjs/docs";
+import * as React from "react";
+
+export class IconExample extends BaseExample<{}> {
+    protected renderExample() {
+        return <div />;
```

Comment:
😄

Reply:
Yeah this is to make things work. I'm not actually sure what to put here given that I can't move the component in yet.

Reply:
can just delete this whole directory, it's not used.